doc/man5/config.pod

```diff
@@ -329,198 +270,158 @@ string for EVP_set_default_properties().
-value will be silently ignored. In certain circumstances such as with
-DNs the same field may occur multiple times. This is usually worked
-around by ignoring any characters before an initial B<.> e.g.
-
- 1.OU="My first OU"
- 2.OU="My Second OU"
+=item B<default_algorithms>
 
-=head1 EXAMPLES
+This sets the default algorithms an ENGINE will supply using the function
+ENGINE_set_default_string().
 
-Here is a sample configuration file using some of the features
-mentioned above.
+=back
 
- # This is the default section.
+All other names are taken to be the name of a ctrl command that is
+sent to the ENGINE, and the value is the argument passed with the command.
+The special value B<EMPTY> means no value is sent with the command.
+For example:
 
- HOME=/temp
- configdir=$ENV::HOME/config
+ [engines]
+ foo = foo_engine
 
- [ section_one ]
+ [foo_engine]
+ dynamic_path = /some/path/fooengine.so
+ some_ctrl = some_value
+ default_algorithms = ALL
+ other_ctrl = EMPTY
+
+=head1 EXAMPLES
+
+This example shows how to using quoting and escaping.
 
- # We are now in section one.
+ # This is the default section.
+ HOME = /temp
+ configdir = $ENV::HOME/config
 
+ [ section_one ]
  # Quotes permit leading and trailing whitespace
  any = " any variable name "
-
  other = A string that can \
  cover several lines \
  by including \\ characters
-
  message = Hello World\n
 
  [ section_two ]
-
  greeting = $section_one::message
 
-This next example shows how to expand environment variables safely.
+This example shows how to expand environment variables safely.
+In this example, the variable B<tempfile> is intended to refer
+to a temporary file, and the environment variable B<TEMP> or
+B<TMP>, if present, specify the directory where the file
+should be put.
+Since the default section is checked if a variable does not
+exist, it is possible to set B<TMP> to default to F</tmp>, and
+B<TEMP> to default to B<TMP>.
 
-Suppose you want a variable called B<tmpfile> to refer to a
-temporary filename. The directory it is placed in can determined by
-the B<TEMP> or B<TMP> environment variables but they may not be
-set to any value at all. If you just include the environment variable
-names and the variable doesn't exist then this will cause an error when
-an attempt is made to load the configuration file. By making use of the
-default section both values can be looked up with B<TEMP> taking
-priority and B</tmp> used if neither is defined:
+ # These two lines must be in the default section.
+ TMP = /tmp
+ TEMP = $ENV::TMP
 
- TMP=/tmp
- # The above value is used if TMP isn't in the environment
- TEMP=$ENV::TMP
- # The above value is used if TEMP isn't in the environment
- tmpfile=${ENV::TEMP}/tmp.filename
+ # This can be used anywhere
+ tmpfile = ${ENV::TEMP}/tmp.filename
 
-Simple OpenSSL library configuration example to enter FIPS mode:
+This example shows how to enforce FIPS mode for the application
```

Comment:
This example uses the deprecated way. Should the `default_properties = fips=yes` be used instead?